Models/GalleryPartItem.cs

```diff
@@ -5,5 +5,6 @@ public class GalleryPartItem {
         public int Type { get; set; }
         public string TypeName { get; set; }
         public string Url { get; set; }
+        public string Thumb { get; set; }

```

Comment:
Ordering of properties within this class is alphabetical so I'd say move `Thumb` to top. When ordering class properties try to make them alphabetical so we have a consistent approach to ordering.